docker-app/qfieldcloud/subscription/models.py

```diff
@@ -819,70 +845,43 @@ def create_subscription(
             # remove microseconds as there will be slight shift with the remote system data
             active_since = active_since.replace(microsecond=0)
 
-        regular_active_since: datetime | None = None
-        if plan.is_trial:
-            assert isinstance(
-                active_since, datetime
-            ), "Creating a trial plan requires `active_since` to be a valid datetime object"
-
-            active_until = active_since + timedelta(days=config.TRIAL_PERIOD_DAYS)
-            logger.info(
-                f"Creating trial subscription from {active_since=} to {active_until=}"
-            )
-            trial_subscription = cls.objects.create(
-                plan=plan,
-                account=account,
-                created_by=created_by,
-                status=plan.initial_subscription_status,
-                active_since=active_since,
-                active_until=active_until,
+        current_time = timezone.now()
+
```

Comment:
```suggestion
        now = timezone.now()
```

I think in pretty much every other case we name the variable `now`.